eng/pipelines/pipeline-generation.yml

```
@@ -145,6 +145,12 @@ jobs:
     condition: ne(variables['TestOptions'],'')
     env:
       PATVAR: $(azuresdk-azure-sdk-devops-pipeline-generation-pat)
+  - script: |
+      $(Pipeline.Workspace)/pipeline-generator/pipeline-generator --organization https://dev.azure.com/azure-sdk --project internal --prefix $(Prefix) --path $(Pipeline.Workspace)/$(RepositoryName)/sdk --endpoint Azure --repository Azure/$(RepositoryName) --convention weekly --agentpool Hosted --branch refs/heads/$(DefaultBranch) --patvar PATVAR --debug $(TestOptions)
+    displayName: 'Generate weekly test pipelines (multi-cloud) for: $(RepositoryName)'
+    condition: ne(variables['TestOptions'],'')
```

```suggestion
    condition: and(succeeded(), ne(variables['TestOptions'],''))
```
Should probably fix the other condition above as well.

---

Do we want to fail fast, or try to create as many pipelines as we can (e.g. still create live pipelines if CI pipelines fail to create, etc.)?

---

I think it is just good practice to stop if something fails or is canceled, in this case I think stopping is the right thing to do.